lib/spack/docs/containers.rst

```diff
@@ -157,17 +167,18 @@ of environments:
        # singularity or anything else that is currently supported
        format: docker
 
-       # Select from a valid list of images
-       base:
-         image: "centos:7"
+       # Sets the base images for the build and final stage
```

Comment:
Additional request: I have the same complaint about this comment that I made in https://github.com/spack/spack/pull/15028#discussion_r419749236

Reply:
Let me know if the current wording seems better